src/functTest/groovy/com/bmuschko/gradle/docker/DockerInfoFunctionalTest.groovy

```diff
@@ -17,4 +17,44 @@ class DockerInfoFunctionalTest extends AbstractFunctionalTest {
         then:
         result.output.contains('Retrieving Docker info.')
     }
+
+    def "Calls onNext when provided"() {
```

Comment:
Can we can combine the expectations of this test with `Passes dockerinfo to onNext`?